docs/guides/cli.md

```diff
@@ -1 +1,174 @@
+- By default, GuideLLM will run a `sweep` of performance evaluations across different request rates, each lasting 120 seconds. The results will be saved to a local directory.
+
+#### 3. Analyze the Results
+
+After the evaluation is completed, GuideLLM will output a summary of the results, including various performance metrics. The results will also be saved to a local directory for further analysis.
+
+The output results will start with a summary of the evaluation, followed by the requests data for each benchmark run. For example, the start of the output will look like the following:
+
+<img alt="Sample GuideLLM benchmark start output" src="https://github.com/neuralmagic/guidellm/blob/main/docs/assets/sample-output-start.png" />
+
+The end of the output will include important performance summary metrics such as request latency, time to first token (TTFT), inter-token latency (ITL), and more:
+
+<img alt="Sample GuideLLM benchmark end output" src="https://github.com/neuralmagic/guidellm/blob/main/docs/assets/sample-output-end.png" />
+
+	
+
+## GuideLLM CLI Details
+### Input Metrics
+The input arguments are split up into 3 sections: 
+
+- **Workload Overview**
+- **Workload Data**
+- **Workload Type**
+
+Once you fill out these arguments and run the command, GuideLLM will run the simulated workload. Note the time it takes to run can be set with <em>max_seconds</em>, but may also depend on the hardware and model. 
+
+#### Workload Overview
+
+This section of input parameters covers what to actually benchmark including the target host location, model, and task. The full list of arguments and their defintions are presented below:
+
+-   **--target** <em>(str, default: localhost with chat completions api for VLLM)</em>: Target for benchmarking 
+    
+	-   optional breakdown args if target isn't specified:
+    
+		-   **--host** <em>(str)</em>: Host URL for benchmarking
+    
+		-   **--port** <em>(str)</em>: Port available for benchmarking
+    
+-   **--backend** <em>(str, default: server_openai [vllm, TGI, llama.cpp, DeepSparse, and many popular servers match this format])</em>: Backend type for benchmarking
+    
+-   **--model** <em>(str, default: auto populated from vllm server)</em>: Model being used for benchmarking, running on the inference server
+        
+-   **--task** <em>(str), optional)</em>: Task to use for benchmarking
+
+-  **--output-path** <em>(str), optional)</em>: Path to save report report to
+
+
+
+#### Workload Data
+
+This section of input parameters covers the data arguments that need to be supplied such as a reference to the dataset and tokenizer. The list of arguments and their defintions are presented below:
+
+-   **--data** <em>(str)</em>: Data file or alias for benchmarking
+
+-   **--data-type** <em>(ENUM, default: emulated; [file, transformers])</em>: The type of data given for benchmarking
+
+-   **--tokenizer** <em>(str)</em>: Tokenizer to use for benchmarking
```

Comment:
curious where you would need or want to customize the tokenizer. Does guidellm tokenize text, or send plaintext?